kqueen/conftest.py

```diff
@@ -23,6 +23,7 @@
 def app():
     """Prepare app."""
     app = create_app()
+    app.testing = True
```

Comment:
Can you comment what is this doing?

Reply:
According to Flask docs it should be set to True in test app, possibly so you can test inside the actual project code if it was ran by test case. Not sure if we actually need it, but should be set to True I guess?